CHANGELOG.md

```diff
@@ -53,6 +54,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 - Fixed UI freezes in Eclipse on bug count decorations update ([#285](https://github.com/spotbugs/spotbugs/issues/285))
 - Bumped log4j from 2.17.1 to 2.17.2 ([#1960](https://github.com/spotbugs/spotbugs/pull/1960))
 - Bumped gson from 2.8.9 to 2.9.0 ([#1960](https://github.com/spotbugs/spotbugs/pull/1966))
+- Fixed detector `ThrowingExcpetions` by omitting cases where throwing `java.lang.Exception` or `java.lang.Throwable` is unavoidable ([#2040](https://github.com/spotbugs/spotbugs/pull/2040))
```

Comment:
Misspelled "ThrowingExceptions"

Reply:
This conversation can be resolved, since the typo was fixed.